.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -22,7 +22,8 @@ body:
     options:
     - Windows
     - macOS
-    - "*nix"
+    - Linux
+    - Other
```

Comment:
Other will prob be those trying to run MultiMC on ARM/aarch devices

Reply:
Maybe specify the architecture. There will eventually be ARM support due to Apple's new M1 devices.

Reply:
Then maybe;

```
    - Windows
    - macOS
    - masOS(M1/ARM)
    - Linux
    - Other
 ```

That would cover those new devices.

Reply:
I was more thinking of just separting OS and architecture.
```yaml
os:
  - Linux
  - Windows
  - MacOS
  - Other

arch:
  - x86_64 (amd64)
  - x86 (i386)
  - ARM64 (M1)
```

Reply:
Though the M1 might be a bit misleading, since ARM64 also applies to other devices of course.

Reply:
I doubt most people know what `x86_64`, `amd64`, `x86` or `i386` mean.
`amd64` is especially misleading as not all `amd64` are from AMD.

Reply:
```
arch:
  - 64 bit (x86_64/amd64)
  - 32 bit (x86/i386/i686)
  - ARM64 (M1)
```

`amd64` & `x86_64` would confuse those poor *doze users :stuck_out_tongue:

Reply:
just leave those four, no reason to over complicate it for one in a million issues

Reply:
Just the OS is fine for now, I think.